tests/bsim/bluetooth/host/att/sequential/tester/src/main.c

```diff
@@ -0,0 +1,715 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+
+#include <zephyr/net/buf.h>
+#include <zephyr/bluetooth/buf.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/hci_raw.h>
+
+#include "common/bt_str.h"
+
+#include "host/conn_internal.h"
+#include "host/l2cap_internal.h"
+
+#include "utils.h"
+#include "bstests.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_tinyhost, LOG_LEVEL_INF);
+
+#define BT_ATT_OP_MTU_REQ 0x02
+#define BT_ATT_OP_MTU_RSP 0x03
+#define BT_ATT_OP_WRITE_REQ 0x12
+#define BT_ATT_OP_WRITE_RSP 0x13
+#define BT_ATT_OP_NOTIFY 0x1b
+#define BT_ATT_OP_INDICATE 0x1d
+#define BT_ATT_OP_CONFIRM 0x1e
+#define BT_ATT_OP_WRITE_CMD 0x52
+#define BT_L2CAP_CID_ATT 0x0004
```

Comment:
Might be nice to align the values here on the same column

Reply:
clang-format'd it

Reply:
Hmm, usually if you add alignment for a few yourself, clang-format will format the rest (or at least my clang-format does this).

Reply:
GH review sucks :/ 
My workflow is I thumbsup/reply as I'm addressing the comments locally, only and after doing all of them do I push.
Sorry about the early notifications. If only we had gerrit...

Reply:
> GH review sucks :/

Indeed

> If only we had gerrit...

Which we used to have, but alas not anymore :( Gitlab would also have been a better alternative than GH IMO.